sim/target-agnostic.mk

```diff
@@ -237,7 +237,6 @@ xsim: $(xsim)
 #########################
 UNITTEST_CONFIG ?= AllUnitTests
 
-rocketchip_dir := $(chipyard_dir)/generators/rocket-chip
```

Comment:
`rocketchip_dir` is defined for `FIRESIM_STANDALONE` in sim/Makefile#L35 .  For Chipyard mode, it ... oops, I guess Chipyard/variables.mk defines `ROCKETCHIP_DIR` and that's why this was here.  I'll push another commit that hoists it into `sim/Makefile` so that it can be overridden by TARGET_PROJECT_MAKEFRAG and won't be broken for Chipyard mode.

Reply:
Okay, now the `rocketchip_dir := $(ROCKETCHIP_DIR)` immediately after including `chipyard/variables.mk` will achieve this but allow itself to be overridden by `TARGET_PROJECT_MAKEFRAG`